scripts/validate-app-templates.js

```diff
@@ -13,6 +13,7 @@ const appTemplates = require('../lib/app-templates');
 const validateAppTemplate = (template, rootTmpDir) => {
   //const appDir = path.resolve(rootTmpDir, template);
   const zapierCmd = path.resolve(__dirname, '../zapier.js');
+  const extraCmd = (template === 'babel') ? ' && rm -rf lib && node_modules/babel-cli/bin/babel.js src --out-dir lib' : '';
```

Comment:
I was waiting on https://github.com/zapier/zapier-platform-example-app-babel/pull/2 to get in. Now I can change that to simply `npm run zapier-build`.